packages/cohort/src/state/postgres/data_store.rs

```diff
@@ -78,7 +83,16 @@ impl DataStore {
     }
 
     pub fn account_from_row(row: &Row) -> BankAccount {
-        row.get::<&str, Json<BankAccount>>("data").0
+        BankAccount {
+            name: row.get::<&str, String>("name"),
+            number: row.get::<&str, String>("number"),
+            version: row.get::<&str, i64>("version") as u64,
+            balance: Money::from_str(
+                row.get::<&str, String>("amount").as_str(),
```

Comment:
`rusty_money` package we use stores the amount as `Decimal`. 

![image](https://github.com/kindredgroup/talos/assets/118979108/8f08daff-bd9a-4a1e-b308-a1763401fb87)

And if we use `bank_accounts.balance` as `DECIMAL` instead of `VARCHAR`, we could use `Money::from_decimal` instead of `Money::from_str. Which is way  simpler and less over head. 

Code of `Money::from_str`
![image](https://github.com/kindredgroup/talos/assets/118979108/c9f8f914-b0b3-45da-bb5f-a29cab1d657d)


vs Code from `Money::from_decimal`
![image](https://github.com/kindredgroup/talos/assets/118979108/cf0c520e-b409-44f0-9ae9-d48f2e0fd361)

Reply:
This is good, but we need to have Decimal instance to use "Money::from_decimal" :)